volumes/php/pm.ini

```diff
@@ -0,0 +1,7 @@
+# Change according to your needs
+# https://docs.nextcloud.com/server/21/admin_manual/installation/server_tuning.html#tune-php-fpm
+[www]
+pm.max_children = 10
+pm.start_servers = 2
+pm.min_spare_servers = 1
+pm.max_spare_servers = 3
```

Comment:
I think that this would be best to stay only at a documentation file because those values are variables and could change depending of quantity of cores at server.

More information here:

https://spot13.com/pmcalculator/

Also, isn't good to version files inside volumes folder because volume sound as a directory that could be deleted if I wish to clean the state of all services.

Reply:
@mateusrovedaa what do you think?

Reply:
I think the same, I think these settings are fine tuning and not everyone will do it. It's not something that's that simple to do and it will vary depending on RAM and CPU, in addition to the consumption of each FPM child process.

It's better to just document it.